### PR TITLE
molecule tests and permissions fix

### PR DESCRIPTION
**Pull request by robled**
* Added 'become' functionality in the event of logging in as non-root
users

* Added molecule/openstack configuration

* Added molecule test for commonly-used role variables

* Added tests/site.yml for commonly-used role variables

### CODE REVIEW

File: tests/roles/firewalld [hunk 1]
```diff
@@ -0,0 +1 @@
+../../
```
Comment (greg-hellings) [inline tests/roles/firewalld]: Is this file necessary when we're running molecule?

File: tests/site.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- hosts: firewalld-servers
```
Comment (greg-hellings) [inline tests/site.yml]: And this?

File: tests/roles/firewalld (hunk 1 above)
Comment (robled) [reply tests/roles/firewalld]: No, I added that when I was running Vagrant.  Should we include this in the event that someone wants to test with Vagrant, or is that redundant with molecule?

File: tests/site.yml (hunk 2 above)
Comment (robled) [reply tests/site.yml]: No, I added that when I was running Vagrant.  Should we include this in the event that someone wants to test with Vagrant, or is that redundant with molecule?
